libkbfs/mdserver_remote.go

```diff
@@ -836,9 +860,68 @@ func (md *MDServerRemote) backgroundRekeyChecker(ctx context.Context) {
 }
 
 // GetKeyBundles implements the MDServer interface for MDServerRemote.
-func (md *MDServerRemote) GetKeyBundles(_ context.Context,
-	wkbID TLFWriterKeyBundleID, rkbID TLFReaderKeyBundleID) (
+func (md *MDServerRemote) GetKeyBundles(ctx context.Context,
+	tlf TlfID, wkbID TLFWriterKeyBundleID, rkbID TLFReaderKeyBundleID) (
 	*TLFWriterKeyBundleV3, *TLFReaderKeyBundleV3, error) {
-	// MDv3 TODO: implement this
-	return nil, nil, nil
+
+	arg := keybase1.GetKeyBundlesArg{
+		FolderID:       tlf.String(),
+		WriterBundleID: wkbID.String(),
+		ReaderBundleID: rkbID.String(),
+	}
+
+	response, err := md.client.GetKeyBundles(ctx, arg)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var wkb *TLFWriterKeyBundleV3
+	if response.WriterBundle.Bundle != nil {
+		if response.WriterBundle.Version != int(SegregatedKeyBundlesVer) {
+			err = fmt.Errorf("Unsupported writer bundle version: %d",
+				response.WriterBundle.Version)
+			return nil, nil, err
+		}
+		wkb := new(TLFWriterKeyBundleV3)
```

Comment:
You're shadowing `wkb` above, which won't be set by this, and thus you'll return `nil` below. Should be `=` instead of `:=`.